sky/templates/gcp-ray.yml.j2

```diff
@@ -306,6 +307,7 @@ setup_commands:
     mkdir -p ~/.ssh; (grep -Pzo -q "Host \*\n  StrictHostKeyChecking no" ~/.ssh/config) || printf "Host *\n  StrictHostKeyChecking no\n" >> ~/.ssh/config;
     python3 -c "from sky.skylet.ray_patches import patch; patch()" || exit 1;
     [ -f /etc/fuse.conf ] && sudo sed -i 's/#user_allow_other/user_allow_other/g' /etc/fuse.conf || (sudo sh -c 'echo "user_allow_other" > /etc/fuse.conf');
+    sudo systemctl stop jupyter > /dev/null 2>&1 || true;
```

Comment:
Is it possible to disable the jupyter service when the VM is starting up instead of doing it in the setup script?
```
metadata:
    startup-script: |
        #! /bin/bash
        systemctl stop jupyter # or maybe `systemctl disable jupyter`
```

Reply:
Just curious, is it possible to remove that `systemd` service in the startup script, so the service will not be started? https://www.linode.com/docs/guides/start-service-at-boot/

Reply:
Did you mean the `.service` file..? It seems the `.service` file is also created after the startup script. I `ls` the `/lib/systemd/system` in the startup script and don't get the `jupyter.service`:
```bash
# startup-script
#! /bin/bash
ls /lib/systemd/system > /tmp/services.txt 2>&1
```
```bash
(base) gcpuser@ray-gcp-cbl-2-head-b9eccdb9-compute:~$ cat /tmp/services.txt | grep jupyter
(base) gcpuser@ray-gcp-cbl-2-head-b9eccdb9-compute:~$ echo $?
1
```
and after launch I SSH into the cluster and double-checked that the `jupyter.service` is indeed inside `/lib/systemd/system`:
```bash
(base) gcpuser@ray-gcp-cbl-2-head-b9eccdb9-compute:~$ ls /lib/systemd/system | grep jupyter
jupyter.service
```

Reply:
Interesting, if that is the case, I guess stopping the Jupyter service in the setup makes sense.